app/controllers/users_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
   end
 
   def edit
-    @user = current_user
+    current_user
```

Comment:
I'm a little unclear why we need to call current_user in the controller if we have it in the view? But I assume it was a test that couldn't pass without it.